### PR TITLE
Added mock liberata profile with active, deleted, on-hold

### PR DESCRIPTION
**Pull request by vigneshpushkaranHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/src/main/resources/application.properties [hunk 1]
```diff
@@ -57,6 +57,8 @@ liberata.oauth2.password=${LIBERATA_OAUTH2_PASSWORD:password}
 liberata.oauth2.base.url=${LIBERATA_OAUTH2_BASE_URL:https://bpacustomerportal.liberata.com/pba/public/api/v2}
 liberata.oauth2.authorize.url=${LIBERATA_OAUTH2_AUTHORIZE_URL:https://bpacustomerportal.liberata.com/pba/public/oauth/authorize}
 liberata.oauth2.token.url=${LIBERATA_OAUTH2_TOKEN_URL:https://bpacustomerportal.liberata.com/pba/public/oauth/token}
+liberata.api.mock=${MOCK_LIBERATA_RESPONSE:false}
+liberat.api.mock.account=pba0001_ACTIVE,pba0002_DELETED,pba0003_ON_HOLD
```
Comment (AyanDuttaHMCTS) [inline api/src/main/resources/application.properties:61]: Can we have Accounts like PBA0001_ACTIVE

File: api/src/test/resources/application-componenttest.properties [hunk 2]
```diff
@@ -4,6 +4,7 @@ spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDri
 spring.datasource.url=jdbc:tc:postgresql:11.0://localhost/features
 
 liberata.api.account.url=http://liberata.com
+liberata.api.mock = true
```
Comment (AyanDuttaHMCTS) [inline api/src/test/resources/application-componenttest.properties:7]: I think we need the PBA Account List here for functional test case

File: model/src/main/java/uk/gov/hmcts/payment/api/service/AccountServiceImpl.java [hunk 3]
```diff
@@ -16,26 +20,43 @@
...
+    private org.springframework.core.env.Environment environment;
 
     @Value("${liberata.api.account.url}")
     private String baseUrl;
 
+    @Value("${liberata.api.mock}")
+    private Boolean mockLiberata;
+
+    @Value("${liberat.api.mock.account}")
+    private String accounts;
```
Comment (AyanDuttaHMCTS) [inline model/src/main/java/uk/gov/hmcts/payment/api/service/AccountServiceImpl.java:36]: renaming accounts to 'mockPbaAccounts' will make more sense

**Review by AyanDuttaHMCTS — CHANGES_REQUESTED** [review]
Changes looks perfect, please refactor the minor comments

**Review by AyanDuttaHMCTS — APPROVED** [review]
Changes looks good now.

**Review by Jennings-HMCTS — APPROVED** [review]

**Pull request closed without merging**